data-import/src/main/resources/folijet/data-import/samples/samples_for_upload/create_action_profile_updateInstance.json

```diff
@@ -2,14 +2,14 @@
   "profile": {
```

Comment:
Hi @SatishkumarPurswani !
We can rename this file to action_profile.json, because it is being used as a template.

Reply:
@SatishkumarPurswani  Yes, change it please.

Reply:
Hi @AivarIUsupov , @VRohach  - Changes done.